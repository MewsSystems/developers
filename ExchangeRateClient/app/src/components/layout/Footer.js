import React from 'react';

export default () => {
  return (
    <footer className="footer">
    <div className="wrapper">
      Copyright &copy; {new Date().getFullYear()} 
      </div>
    </footer>
  );
};
