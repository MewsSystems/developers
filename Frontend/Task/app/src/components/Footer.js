import React from 'react';

const Footer = () => (
  <footer className="page-footer font-small blue pt-4 bg-dark text-white">
    <div className="container-fluid text-center text-md-left">
      <div className="row">
        <div className="col-md-6 mt-md-0 mt-3">
          <h5 className="text-uppercase">Lukáš Kratochvíl</h5>
          <p><a href="mailto:kratochvil.lukas9@gmail.com">kratochvil.lukas9@gmail.com</a></p>
        </div>
        <hr className="clearfix w-100 d-md-none pb-3" />
        <div className="col-md-6 mb-md-0 mb-6">
          <h5 className="text-uppercase">FindMe</h5>
          <ul className="list-unstyled">
            <li>
              <a href="https://www.linkedin.com/in/progresak/">LinkedIn</a>
            </li>
            <li>
              <a href="https://github.com/progresak">GitHub</a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
);

export default Footer;
