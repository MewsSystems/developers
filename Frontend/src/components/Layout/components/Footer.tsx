import { COLORS } from 'constants/colors'
import React from 'react'
import styled from 'styled-components'
import { useTranslation } from 'react-i18next'

export const StyledFooter = styled.div`
  padding: 1rem 2rem;
  color: ${COLORS.WHITE};
  background: ${COLORS.DARK_GRAY};
`

export const Footer: React.FC = () => {
  const { t } = useTranslation()
  const year = new Date().getFullYear()

  return <StyledFooter>{t('footer.copyright', { year })}</StyledFooter>
}
