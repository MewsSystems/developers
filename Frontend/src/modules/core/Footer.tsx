import React from 'react';

interface Props {
    className?: string,
}

export default function Footer({ className }: Props) {
    return (
        <footer className={className}>
            Made by
            {' '}
            <a
                href={'https://github.com/fabulator'}
                target={'_blank'}
                rel={'noopener noreferrer'}
            >
                Michal
            </a>
            . Designed in Czech Republic. Manufactured in Czech Republic.
        </footer>
    );
}
