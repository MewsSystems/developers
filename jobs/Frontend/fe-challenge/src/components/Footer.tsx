import Logo from '@/components/Logo';

const Footer = () => {
  return (
    <footer className="border-t h-24 border-slate-300/30 bg-gray-900 px-6">
      <div className="container mx-auto h-full flex items-center">
        <Logo className="w-[120px] text-white" />
      </div>
    </footer>
  );
};

export default Footer;
