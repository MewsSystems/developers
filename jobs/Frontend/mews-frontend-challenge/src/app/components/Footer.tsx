export function Footer() {
  return (
    <footer className="flex items-center justify-center bg-slate-900 p-8 text-white">
      Ricardo Dominguez Alvarez - 2024
    </footer>
  );
}
