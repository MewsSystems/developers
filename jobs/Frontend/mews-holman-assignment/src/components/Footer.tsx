import Box from "@mui/material/Box";

const Footer = () => {
  return (
    <Box
      sx={{
        py: 3,
        px: 2,
        mt: "auto",
        backgroundColor: "primary.main",
      }}
    ></Box>
  );
};

export default Footer;
