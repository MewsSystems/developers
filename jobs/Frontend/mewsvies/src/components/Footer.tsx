export const Footer = () => {
    return (
        <footer className="bottom-0 left-0 w-full bg-gray-800 text-white py-4">
            <div className="container mx-auto px-4 flex flex-col items-center">
                <div className="mb-4">
                    <span>by:</span>
                    <a
                        href="https://davidportilla.com"
                        target="_blank"
                        rel="noreferrer"
                        className="ml-1 font-bold"
                    >
                        David Portilla
                    </a>
                </div>
                <ul className="flex">
                    <li className="mr-4">
                        <a
                            href="https://www.linkedin.com/in/davidportilla/"
                            target="_blank"
                            rel="noreferrer"
                            className="font-bold"
                        >
                            Linkedin
                        </a>
                    </li>
                    <li>
                        <a
                            href="https://github.com/david-portilla"
                            target="_blank"
                            rel="noreferrer"
                            className="font-bold"
                        >
                            Github
                        </a>
                    </li>
                </ul>
            </div>
        </footer>
    );
};
