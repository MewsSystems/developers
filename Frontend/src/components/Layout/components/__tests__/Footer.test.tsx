import { Footer } from '../Footer'
import React from 'react'
import { shallow } from 'enzyme'

describe('Test Footer', () => {
  it('Tests snapshot', () => {
    expect(shallow(<Footer />)).toMatchSnapshot()
  })
})
