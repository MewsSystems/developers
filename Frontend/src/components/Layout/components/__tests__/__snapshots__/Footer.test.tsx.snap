// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer Tests snapshot 1`] = `
<styled.div>
  footer.copyright
</styled.div>
`;
